cypress.config.ts

```diff
@@ -1,3 +1,5 @@
+// cypress SHOULD be a dev dependency
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
comments should be written like this 
`// eslint-disable-next-line import/no-extraneous-dependencies -- cypress SHOULD be a dev dependency`